README.md

```diff
@@ -348,46 +348,6 @@ Environment variable `DOCKER_CLUSTER_PROVIDER (default: none)` can be set for th
 _NB1: For now, only Docker Swarm is supported, and only autorestart takes the cluster into account. More complete cluster support will be added in the future._
 _NB2: Since running an arbitrary command on all nodes of a service breaks the service abstraction, autocmd is not supported in Docker Swarm mode._
 
-### Run a custom deploy hook script
```

Comment:
Why do you remove this section ?

Reply:
It is duplication. Look at current readme.md at main page.

Reply:
Ah, right! Refactoring this big Readme in a Wiki is on my todo list.